src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -59,7 +62,8 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
 
             this.errorListInfoBarController = this.extensionHost.GetService<IErrorListInfoBarController>();
             this.errorListInfoBarController.AssertLocalServiceIsNotNull();
-            this.errorListInfoBarController.Refresh();
```

Comment:
Refreshing the error list at this point causes a deadlock.
TODO: check whether the gold bar is still correctly displayed, or whether we need to trigger the refresh some other way.